### PR TITLE
Fix Zephyr Firmware Update Example Walkthrough

### PR DESCRIPTION
**Pull request by MarkoPura**
As John highlighted in the [forum post](https://forum.golioth.io/t/where-is-app-update-hex-equivalent-used-for-ota-in-sdk-0-15-0/991/3), the Zephyr Firmware Update Example Walkthrough didn't fully capture the changes in the OTA update process introduced with the addition of sysbuild in Firmware SDK v0.15.0.

I updated the OTA process to align with the description in the `fw_update` example and replaced screenshots to reflect the firmware image name and version as mentioned in the text.

### CODE REVIEW

File: docs/firmware/golioth-firmware-sdk/firmware-upgrade/fw-update-zephyr.md [hunk 1]
```diff
@@ -55,17 +55,17 @@ west build -b nrf9160dk/nrf9160/ns --sysbuild examples/zephyr/fw_update
...
 
-Now update the firmware version in the project `prj.conf` file:
+Now update the firmware version in the project `VERSION` file:
 
 ```cfg
-CONFIG_MCUBOOT_IMGTOOL_SIGN_VERSION="1.2.4"
+VERSION_MAJOR = 1
+VERSION_MINOR = 2
+PATCHLEVEL = 4
 ```
```
Comment (trond-snekvik) [inline docs/firmware/golioth-firmware-sdk/firmware-upgrade/fw-update-zephyr.md:69]: Perhaps we should have the old way in a [tab](https://docusaurus.io/docs/markdown-features/tabs) here and below for some time for users that aren't on the latest SDK?

**Review by trond-snekvik — COMMENTED** [review]
Thanks Marko, I just have one comment🙂

File: docs/firmware/golioth-firmware-sdk/firmware-upgrade/fw-update-zephyr.md (hunk 1 above)
Comment (MarkoPura) [reply docs/firmware/golioth-firmware-sdk/firmware-upgrade/fw-update-zephyr.md:69]: Thank you @trond-snekvik, makes sense!

Comment (MarkoPura) [reply docs/firmware/golioth-firmware-sdk/firmware-upgrade/fw-update-zephyr.md]: @trond-snekvik, added tabs and described how to change the firmware version number with SDK 0.15.0+  and with versions earlier than v0.15.0.

**Review by trond-snekvik — APPROVED** [review]
Nice!

**Pull request merged**